### ISSUE TITLE
"Clear" text for search should be able to be configured

### ISSUE THREAD

**Issue by paul-grant-hs** [ealush/emoji-picker-react#476]
The app in which we use this library is internationalized and uses many languages; the text for the `aria-label` and the `title` should be able to be passed in via configuration.

If you agree, I can make a PR to do this through the `useConfig` the same way the `placeholder` text is set.

**Comment by ealush** [comment]
Closing as completed: this is configurable via the `searchClearButtonLabel` prop, which sets both the aria-label and title of the search clear button.
